packages/expo/src/Svg.web.ts

```diff
@@ -0,0 +1,57 @@
+import { createElement } from 'react';
+import PropTypes from 'prop-types';
+
+function generate(name, type) {
```

Comment:
```suggestion
function generate(componentName, tagName) {
```

In this case we're creating only SVG elements (as opposed to non-primitive React elements) with `createElement` so `tagName` for the second parameter works here. And distinguishing the two types of names might be helpful.